cdap-gateway/src/main/java/io/cdap/cdap/gateway/router/RouterPathLookup.java

```diff
@@ -196,15 +193,10 @@ private RouteDestination getV3RoutingService(String [] uriParts, AllowedMethod r
       // /v3/namespaces/{namespace-id}/data/datasets/{name}/properties
       // /v3/namespaces/{namespace-id}/data/datasets/{name}/admin/{method}
       return DATASET_MANAGER;
-    } else if (beginsWith(uriParts, "v3", "support", "bundle")) {
-      //Support Bundle Handler Path /v3/support/bundle
+    } else if (beginsWith(uriParts, "v3", "support", "bundle")
+      || beginsWith(uriParts, "v3", "support", "bundles")) {
```

Comment:
We should just use /support/bundles everywhere, that's what REST guidelines say.